staff/views/orgs.py

```diff
@@ -1,22 +1,57 @@
+from django.contrib import messages
+from django.db.models.functions import Lower
+from django.shortcuts import get_object_or_404, redirect
 from django.utils.decorators import method_decorator
-from django.views.generic import DetailView, ListView, UpdateView
+from django.views.generic import FormView, ListView, UpdateView, View
 
 from jobserver.authorization import CoreDeveloper
 from jobserver.authorization.decorators import require_role
-from jobserver.models import Org
+from jobserver.models import Org, OrgMembership, User
+
+from ..forms import OrgAddMemberForm
 
 
 @method_decorator(require_role(CoreDeveloper), name="dispatch")
-class OrgDetail(DetailView):
-    model = Org
+class OrgDetail(FormView):
+    form_class = OrgAddMemberForm
     template_name = "staff/org_detail.html"
 
+    def dispatch(self, request, *args, **kwargs):
+        self.object = get_object_or_404(Org, slug=self.kwargs["slug"])
+
+        return super().dispatch(request, *args, **kwargs)
+
+    def form_valid(self, form):
+        for user in form.cleaned_data["users"]:
+            self.object.memberships.create(user_id=user, created_by=self.request.user)
```

Comment:
Could we end up with multiple memberships for a single user here?

Reply:
No, they're unique on Org & User so we should get an error then, but I'll also trim the available orgs down so we only see ones without a membership 👍